cmake/modules/SearchInstalledSoftware.cmake

```
@@ -1872,8 +1879,14 @@ if (roofit_multiprocess)
   endif()
 
   if(builtin_cppzmq)
-    list(APPEND ROOT_BUILTINS cppzmq)
-    add_subdirectory(builtins/zeromq/cppzmq)
+    ROOT_CHECK_CONNECTION("builtin_cppzmq=OFF")
+    if(NO_CONNECTION)
+      message(STATUS "No internet connection, disabling the `builtin_cppzmq` option")
+      set(builtin_cppzmq OFF CACHE BOOL "Disabled because there is no internet connection" FORCE)
```

```suggestion
      message(STATUS "No internet connection, disabling the `builtin_cppzmq` and `roofit_multiprocess` options")
      set(builtin_cppzmq OFF CACHE BOOL "Disabled because there is no internet connection" FORCE)
      set(roofit_multiprocess OFF CACHE BOOL "Disabled because there is no internet connection" FORCE)
```

I guess in that case we need to switch off RooFit multiprocessing too? It needs both zeromq and cppmq

---

OK, fine